en/user_guide/node_configuration.rst

```diff
@@ -0,0 +1,105 @@
+Node configuration
+==================
+
+Here are the available options in the node configuration. Some of them are required
+in order to build a node and must therefore be provided.
+
+Page (required)
+---------------
+
+The page attribute is the name of the node, it is also used to build the url pattern.
```

Comment:
I think it is important to say that contribute the node name will generate the node url pattern